packages/react-examples/src/react/Nav/Nav.Wrapped.Example.tsx

```diff
@@ -0,0 +1,91 @@
+import * as React from 'react';
+import { Nav, INavStyles, INavLinkGroup } from '@fluentui/react/lib/Nav';
+
+const navStyles: Partial<INavStyles> = {
+  root: {
+    width: 208,
+    height: 350,
+    boxSizing: 'border-box',
+    border: '1px solid #eee',
+    overflowY: 'auto',
+  },
+  link: {
```

Comment:
Can you add a comment calling out that this is the important part for accessibility?